<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageNav</class>
 <widget class="QWidget" name="PageNav">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxNavArea">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="layoutNavBar"/>
      </item>
      <item>
       <widget class="NavButton" name="buttonPage4">
        <property name="text">
         <string>Page4</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="routeView">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page1">
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>75</x>
         <y>125</y>
         <width>74</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>CheckBox</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page2">
      <widget class="QCommandLinkButton" name="commandLinkButton">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>125</y>
         <width>186</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>CommandLinkButton</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page3">
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>115</y>
         <width>88</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>RadioButton</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page4">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavButton</class>
   <extends>QPushButton</extends>
   <header location="global">mvvmkit.navigations.ButtonNav.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
